contentcuration/contentcuration/frontend/channelEdit/pages/StagingTreeView/index.spec.js

```diff
@@ -348,7 +348,7 @@ describe('StagingTreeView', () => {
       it('redirects to a topic detail page on info button click', () => {
         expect(wrapper.vm.$router.currentRoute.name).toBeUndefined();
 
-        getInfoBtn(topic).trigger('click');
```

Comment:
Any idea why `trigger` has stopped working? I haven't figured it out myself.

Reply:
It's because it's now an IconButton, so triggering a click somehow isn't getting passed back. You have to trigger it on the nested `[data-test="button"] .v-btn` instead. I think it has to do with v-btn not being the root element as it's wrapped in a tool-tip element